src/ergw_aaa_static.erl

```diff
@@ -107,7 +107,7 @@ handle_response({API, _}, #{'Result-Code' := Code}, Session, Events, State) ->
 handle_response(Procedure, #{'Result-Code' := Code}, Session, Events, State) ->
     {{fail, Code}, Session, [{stop, {Procedure, peer_reject}} | Events], State};
 handle_response(_Procedure, Response, Session, Events, State) ->
-    ?LOG(error, "Response: ~p", [Response]),
+    ?LOG(debug, "unexpected response: ~p", [Response]),
```

Comment:
Would not a log level of warning be better here?

Reply:
I'd agree to that 🤔

Reply:
i do not: https://www.erlang.org/doc/apps/kernel/logger_chapter.html#log-level

the audience of the logs is a user of software incorporating ergw_aaa. some external party floods your instance with garbage, the garbage is properly NOT handled (thus dropped). what is the error? what is the call to action to solve that error?